uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletDefinitionForm.java

```diff
@@ -53,8 +58,9 @@
 public class PortletDefinitionForm implements Serializable {
 
     private static final String FRAMEWORK_PORTLET_URL = "/uPortal";
+	private static final FastDateFormat dateFormat = FastDateFormat.getInstance("M/d/yyyy");
```

Comment:
The date format string should be internationalized. Read from properties file? Not sure how we have internationalization set up.